hack/self-signed-ca.sh

```diff
@@ -117,9 +117,9 @@ echo "Encoded CA:"
 echo -e "${caBundle} \n"
 
 # Patch CA Certificate to webhooks
-mutatingPatchString='[{"op": "replace", "path": "/webhooks/0/clientConfig/caBundle", "value":"{{CA_BUNDLE}}"}, {"op": "replace", "path": "/webhooks/1/clientConfig/caBundle", "value":"{{CA_BUNDLE}}"}]'
+mutatingPatchString='[{"op": "replace", "path": "/webhooks/0/clientConfig/caBundle", "value":"{{CA_BUNDLE}}"}, {"op": "replace", "path": "/webhooks/1/clientConfig/caBundle", "value":"{{CA_BUNDLE}}"}, {"op": "replace", "path": "/webhooks/2/clientConfig/caBundle", "value":"{{CA_BUNDLE}}"}]'
```

Comment:
looks like it was causing the issue after we added v1beta1, not sure if there is better way to do this as I can imagine it is going to be broken again as we are deprecating v1alpha2.

Reply:
maybe we can get webhooks in each configuration using kubectl and then build the patch string using webhook counts

Reply:
right need some way to get the count of the webhooks